src/com/pentalog/pentastagiu/homework2/shop/Basket.java

```diff
@@ -0,0 +1,52 @@
+package com.pentalog.pentastagiu.homework2.shop;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Basket {
+    private static int totalPrice;
```

Comment:
nu este corect sa folosesti un field static: field static inseamna ca valoarea va fi comuna tuturor obiectelor de tip basket, ceea ce nu este cazul(poti face un test in care creezi 2 cosuri, adaugi produse cand in unul, cand in altul si verifici apoi care este totalPrice pentru fiecare obiect in parte)

aceeasi observatie si pentru urmatoarele fielduri, nu iti permit sa lucrezi cu mai multe cosuri independente